src/System.Data.SqlClient/src/System/Data/Common/AdapterUtil.cs

```diff
@@ -0,0 +1,1150 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlTypes;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Security;
+using System.Data.SqlClient;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Res = System.SR;
+using System.Reflection;
+
+namespace System
+{
+    internal static partial class SR
```

Comment:
We are using this class as Res "using Res=System.SR;" 
This was done to maintain the same class names in the classes using resources.
The underlying mechanism is same as the corefx mechanism.